<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>1201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>841</width>
      <height>1035</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="container">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="header">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QComboBox" name="nbr_colors">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:15px;padding-left:10;</string>
         </property>
         <property name="currentIndex">
          <number>3</number>
         </property>
         <item>
          <property name="text">
           <string>3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="size_entry">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font-size:15px;padding-left:10;</string>
         </property>
         <property name="currentText">
          <string>14 x 14</string>
         </property>
         <property name="currentIndex">
          <number>8</number>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string extracomment="3">3 x 3</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string extracomment="4">4 x 4</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>5 x 5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>6 x 6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>7 x 7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>8 x 8</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>9 x 9</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>10 x 10</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>14 x 14</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>18 x 18</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>22 x 22</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>26 x 26</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>30 x 30</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>50 x 50</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>100 x 100</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>150 x 150</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>200 x 200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>300 x 300</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>400 x 400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>500 x 500</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>750 x 750</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>1000 x 1000</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_count">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="create">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Create</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="restart">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Restart</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>2</width>
         <height>800</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>800</height>
        </size>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::DefaultContextMenu</enum>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="accessibleName">
        <string/>
       </property>
       <property name="accessibleDescription">
        <string/>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>1</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAsNeeded</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="autoScroll">
        <bool>false</bool>
       </property>
       <property name="autoScrollMargin">
        <number>0</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>false</bool>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::NoDragDrop</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::NoSelection</enum>
       </property>
       <property name="verticalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="horizontalScrollMode">
        <enum>QAbstractItemView::ScrollPerPixel</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>false</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="verticalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="naif_row">
       <item>
        <widget class="QPushButton" name="dumb_h">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play dumb</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="brute_force_h">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play Brute Force</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="glouton_row">
       <item>
        <widget class="QPushButton" name="glouton_h_1">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play Glouton 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="glouton_h_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play Glouton 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="glouton_h_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play Glouton 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="deep1_h">
         <property name="text">
          <string>Play Deep 1</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deep2_h">
         <property name="text">
          <string>Play Deep 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deep3_h">
         <property name="text">
          <string>Play Deep 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="advanced_row">
       <item>
        <widget class="QPushButton" name="tabu_h">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>Play Tabu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="dijkstra_h">
         <property name="text">
          <string>Play Multiple Dijkstra</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
